packages/core/src/components/Table/Head/HeadCell/HeadCell.styled.tsx

```diff
@@ -64,6 +68,9 @@ export const HeadCellStyled = styled.th<HeadCellStyledProps>`
     position: ${({ frozen }) => (frozen ? 'sticky' : 'relative')};
     cursor: ${({ isRowActionCell }) => isRowActionCell && 'default'};
     padding: ${getHeadCellPadding};
+    ${({theme}) => theme.table.header.fontFamily && getFontFamily(theme.table.header.fontFamily)}
```

Comment:
@mbetances805  If you can use `fontVarint` like mentioned in below comment then you can use `getFontStyle` from `@medly-components/utils`  like below
 `${({ theme }) => getFontStyle({ theme, fontVariant: theme.table.header.fontVariant })}`